commonlibktx/src/androidTest/java/com/crisolutions/commonlibraryktx/ExtensionTests.kt

```diff
@@ -34,9 +35,7 @@ class ExtensionTests {
 
         val music = "Jazz"
 
-        assertBlock {
-            music.transform { it.toUpperCase() } == "JAZZ"
-        }
+        assertEquals(music.transform { it.toUpperCase() }, "JAZZ")
```

Comment:
Swap order of parameters to match method signature.

Reply:
Swapped.